ibex_demo_system.core

```diff
@@ -107,8 +107,7 @@ targets:
     toplevel: top_cw305
     tools:
       vivado:
-        part: "xc7a100tftg256-2" # default to a100 part
```

Comment:
I think we should keep the default to `a100` until we officially decide to move to a different board.

Reply:
This is a mistake, I restored the original file